Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ur20-modbus"
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
Would you mind to remove this version bump?
I'd like to do this in a separate release commit.

Reply:
Sure, done.